packages/astro/components/WelcomeScreen.astro

```diff
@@ -0,0 +1,172 @@
+---
```

Comment:
I think moving the original design here was smart for keeping this PR reviewable! Still, I recall that @FredKSchott wanted to use this as an opportunity to explore a new, more expressive design.

May become a blocker to make the new starter more of a _feature_ we promote. Still on the fence whether merging this change in code structure without a redesign is a good idea.

Reply:
For what it's worth, this is on the next branch, so it would only affect new projects using the beta. Even if we merge this right now, there's plenty of time to make the new design before the actual release.